### ISSUE TITLE
Feature Request: STM32F1 support

### ISSUE THREAD

**Issue by rljacobson** [IntergatedCircuits/STM32_XPD#2]
The cheap [STM32F103C8T6 "Blue Pill" dev board](https://maker.pro/arduino/tutorial/how-to-program-the-stm32-blue-pill-with-arduino-ide) is popular in the hobbyist community because it can be purchased on eBay for < $2. It would be nice to have the XPD library for this device.

**Comment by benedekkupper** [comment]
Yes, it would be nice to have. Me personally have no interest in it though (in my opinion F1 is pretty much NRND), so if it's up to me, it will never happen.

**Comment by rljacobson** [comment]
@IntergatedCircuits I totally understand! Feel free to close this issue if you'd like, or if you want to leave it open to be visible to potential contributors, that's fine to. Thanks!
